workflow/artifacts/git/git.go

```diff
@@ -31,7 +32,18 @@ type ArtifactDriver struct {
 
 var _ common.ArtifactDriver = &ArtifactDriver{}
 
-func (g *ArtifactDriver) auth() (func(), transport.AuthMethod, []string, error) {
+var sshURLRegex = regexp.MustCompile("^(ssh://)?([^/:]*?)@[^@]+$")
+
+func GetUser(url string) (string, bool) {
```

Comment:
Bool is always ignored, so you can remove it

Reply:
remove ignored bool output